web/scripts/app.js

```diff
@@ -201,6 +202,23 @@ function redraw() {
 }
 
 
+function _search_from(query, rowid) {
+    var data = grid.getData();
+    var numrows = data.getLength();
+    var next = false;
+
+    for (var i = rowid + 1; i < numrows; ++i) {
+        var row = data.getItem(i);
+        if (row.searchname == query || row.searchname.indexOf(query) >= 0) {
+            rowid = i;
+            next = true;
+            break;
```

Comment:
Since this logic is in a helper function that returns the row index, the above three lines can be replaced with just `return i;`, and the variables `rowid` and `next` may be removed.